docs/pages/api-reference/liveblocks-react.mdx

```diff
@@ -592,12 +592,20 @@ return `null`.
 Returns a callback that lets you broadcast custom events to other users in the
 room.
 
-```ts
+```ts highlight="3-5"
 import { useBroadcastEvent } from "./liveblocks.config";
 
+// On client A
 const broadcast = useBroadcastEvent();
-
 broadcast({ type: "EMOJI", emoji: "🔥" });
+
+// On client B
+useEventListener(({ event, user }) => {
```

Comment:
```suggestion
useEventListener(({ event, user, connectionId }) => {
```

Even though `user.connectionId` is the same, shouldn't we still show every option? 

Though if we're deprecating `{ connectionId }` soon, we should add a deprecation message to the comments in the package.

Reply:
I don't think we want to (or should) deprecate the `connectionId`. There still is a niche use case for it: if it's `>= 0`, but `user` is still `null`, you could know it's not a server event, but the user is not yet (or no longer) in the room. Depending on the app you may want to ignore those events, yet not ignore server events.

Reply:
You convinced me that it's better to should all the available fields in the example though — will add 👍